src/xdocs-examples/resources/com/puppycrawl/tools/checkstyle/checks/javadoc/javadocpackage/legacy/Example2.java

```diff
@@ -0,0 +1,14 @@
+/*xml
+<module name="Checker">
+  <module name="JavadocPackage">
+    <property name="allowLegacy" value="true"/>
+  </module>
+</module>
+*/
+// xdoc section -- start
+package com.puppycrawl.tools.checkstyle.checks.javadoc.javadocpackage.legacy;
+// OK as package.html file is present in directory
+public class Example2 {
+  String str="some content";
+}
```

Comment:
`public class Example2 { }`

For all, to let people easily see that class content doesn't matter

Reply:
Done